src/parcsr_mv/par_csr_matop.c

```diff
@@ -322,6 +322,14 @@ hypre_ParMatmul_RowSizes( HYPRE_MemoryLocation memory_location,
    *C_diag_size = (*C_diag_i)[num_rows_diag_A];
    *C_offd_size = (*C_offd_i)[num_rows_diag_A];
 
+#ifdef HYPRE_DEBUG
+   for (HYPRE_Int i = 0; i < num_rows_diag_A; i++)
```

Comment:
Should avoid C++ style for-loop declaration for `i`.

Reply:
Hi Ruipeng, This is [C99](https://en.cppreference.com/w/c/language/for), but I can definitely change in order to be aligned to the style we use in hypre.

Reply:
I don't think we use this style in hypre. Maybe just define `HYPRE_Int i` outside?

Reply:
Yes, will do it :)